spacy/sklearn.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+from sklearn.base import BaseEstimator, TransformerMixin
+import spacy
+
+
+__all__ = [
+    'SpacyTransformer',
+    'DOC', 'VECTOR', 'SENTENCES', 'ENTITIES', 'NOUN_CHUNKS',
+    'SENTIMENT', 'TOKENS'
+    ]
+
+
+DOC = 'self'
+VECTOR = 'vector'
+SENTENCES = 'sents'
+ENTITIES = 'ents'
+NOUN_CHUNKS = 'noun_chunks'
+SENTIMENT = 'sentiment'
+TOKENS = 'tokens'
+
+
+class SpacyTransformer(BaseEstimator, TransformerMixin):
+
+    def __init__(self,
+                 model=spacy.en.English(),
```

Comment:
Assigning the model as a keyword argument seems quite problematic here. Was this a deliberate choice?

(In case you're not familiar with this particular Python gotcha: default values for function arguments are only assigned once, so all invocations of the function use the same reference. If you mutate the value, you're quietly changing state between function calls, giving results that are often unexpected.)